packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/styles.scss

```diff
@@ -1,26 +1,32 @@
 // Import
-@import "../../styles/variables/variables";
+@import '../../styles/variables/variables';
 
-.leftMenuLinkContainer { /* stylelint-ignore */
-  padding-top: .6rem;
-  padding-bottom: 10.2rem; // LeftMenuFooter height
+.leftMenuLinkContainer {
+  /* stylelint-ignore */
+  padding-top: 0.6rem;
   position: absolute;
   top: 60px;
   right: 0;
   bottom: 0;
   left: 0;
   overflow-y: auto;
+  height: calc(
+    100vh - (60px + 10.2rem)
+  ); // we know we are 10.2 rems up from the bottom footer and 60px down
```

Comment:
Great, could you just set sizes as rem and remove comment ?

`height: calc(100vh - (0.6rem + 10.2rem));`